LICENSE.md

```diff
@@ -81,6 +81,7 @@ The icons `assets/qr-icon*.svg` (used in `src/icons/icon-small-*.svg`) are made
 * [`popup/img/error-white.svg`](popup/img/error-white.svg) by Mozilla, license: [MPL v2.0](https://www.mozilla.org/en-US/MPL/2.0/)
 * [`popup/img/info-dark.svg`](popup/img/info-dark.svg) by Mozilla, license: [MPL v2.0](https://www.mozilla.org/en-US/MPL/2.0/)
 * [`popup/img/info-light.svg`](popup/img/info-light.svg) by Mozilla, license: [MPL v2.0](https://www.mozilla.org/en-US/MPL/2.0/)
+* [`icons/photon/open-in-new-16.svg`](icons/photon/open-in-new-16.svg) by Mozilla, license: [MPL v2.0](https://www.mozilla.org/en-US/MPL/2.0/)
```

Comment:
There is no need for a phaeton dir. All current icons are from Photon anyway…

So either move all of them into this dir – or well… just put the new one is this dir.

Reply:
I just didn't put the icon in the `popup/img/` directory because this icon is not really related to the popup but rather to the options/settings. But maybe it will be needed later so I will move it into this folder...